web/templates/login.html

```diff
@@ -0,0 +1,61 @@
+{{ template "header.html" . }}
```

Comment:
For reference, [this](https://github.com/dexidp/dex/blob/master/web/templates/login.html) is the `login.html` template from dex's repo.

The JS executing the connector filtering logic is contained within the `<script />` tags. For ease of review, I haven't minimized it.